second-edition/src/ch12-02-reading-a-file.md

```diff
@@ -1,14 +1,14 @@
-## Reading a File
+## Leyendo un Archivo
 
-Now we’ll add functionality to read the file that is specified in the
-`filename` command line argument. First, we need a sample file to test it with:
-the best kind of file to use to make sure `minigrep` is working is one with a
-small amount of text over multiple lines with some repeated words. Listing 12-3
-has an Emily Dickinson poem that will work well! Create a file called
-*poem.txt* at the root level of your project, and enter the poem “I’m Nobody!
-Who are you?”
+Ahora añadiremos funcionalidad para leer el archivo especificado en el argumento de la 
```

Comment:
Please revise that spanish punctuation signs ! need also an opening matching one :)

Reply:
all fixed 👍